### DISCUSSION TITLE
saas using sqlite has db migrate has error with  NaiveDateTime keyword

### DISCUSSION THREAD

**Discussion by tjyang** [loco-rs/loco#349] [Q&A] [answered]
I am hoping simple sqlite db can be used to learn loco app, Without getting into PostgreSQL or MySQL.

-  [This getting started doc](https://loco.rs/docs/getting-started/tour/)  followed but switched over to sqlite db.
- error message when running cargo test or cargo loco db migrate
![image](https://github.com/loco-rs/loco/assets/378638/ed4e5138-5f79-4f34-b249-0e1bae2a0095)

**Comment by tjyang** [comment]
@kaplanelad , 
I vaguely remember you mentioned sqlite is not 100% supported due to this NaiveDateTime error. 
Can you confirm please ?

**Reply by kaplanelad** [reply, marked as answer]
It is not supported 100% by sea_orm. 
See sea_orm open [issue](https://github.com/SeaQL/sea-orm/issues/924)

**Comment by tjyang** [comment]
Thanks for providing the link to root cause. I am closing this discussion.

**Comment by tjyang** [comment]
here is another info at https://github.com/loco-rs/loco/pull/210
